exercises/twofer/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+name = "twofer"
+version = "0.1.0"
```

Comment:
This should match the [version in `canonical-data.json`](https://github.com/exercism/problem-specifications/blob/master/exercises/two-fer/canonical-data.json#L3).